### ISSUE TITLE
Account Settings link in My account dashboard page not demarcated properly in high contrast mode

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#934]
In a Macbook, turn on Increase Contrast (System Preferences -> Accessibility -> Display -> 'Increase Contrast Mode'). Then go to your Firefox browser and go to Settings -> General -> Language & Appearance -> Colors, and then click the 'Only with High Contrast themes' option on 'Override the colors specified by the page with your selections above'.

On a Windows machine, in Firefox, go to Settings -> Vision -> Use High Contrast and set 'Turn on high contrast' to On.

Now go to Firefox and log in to the public portal. There will be a welcome message when you sign in. However, in High Contrast mode, the message doesn't have a border and it will be difficult for user to identify it as such. Also, the Close button doesn't have a border so it will be difficult for user to identify it as such.

NOTE: I was able to replicate this with Chrome via the High Contrast extension.

Two images attached: one regular and one with high contrast mode enabled.

![welcome_back_image_normal](https://user-images.githubusercontent.com/394908/147710739-fc54a3d0-6b76-4daf-9d8b-38ff90ca6713.png)
<img width="1233" alt="Screen Shot 2021-12-29 at 6 32 58 PM" src="https://user-images.githubusercontent.com/394908/147710745-c16ffc85-28cc-4e20-bf1f-dbc113d90b34.png">
